### ISSUE TITLE
Add keyboard shortcuts for yes/no/details

### ISSUE THREAD

**Issue by joeduffy** [pulumi/pulumi#1364]
We interactively prompt during a `pulumi update` and family, after doing the preview, to confirm that a user would like to proceed.  Right now this requires up/down/enter sequences to navigate.

It would be nicer if you could use single letter keystrokes to proceed.

I would recommend something like the following

```
Do you want to perform this update?
  (Y)es
> (n)o
  (m)ore details
```

where the first letter in parenthesis is used to take that particular action.

**Comment by Tirke** [comment]
Are we not limited by [survey](https://github.com/AlecAivazis/survey) capabilities here ? Updating survey to a version ≥ 1.5 will provide optional keyboard filtering but I didn't see a way to provide additional key listeners to survey prompt. I've also looked at other existing interactive prompts and I didn't find one who clearly advertise the feature we want here. 

Maybe I'm missing something ? 😞

**Comment by ellismg** [comment]
> Maybe I'm missing something ? :disappointed:

I do not believe you are @Tirke.  This would be a slight quality of life improvement, but not the most important thing we can do be doing right now.  If someone wants to try to tackle this, more power to them, but I do think it's going to be a bit of a chore since the library we use today does not support this.

**Comment by nesl247** [comment]
https://github.com/AlecAivazis/survey/issues/261 is the upstream issue for this it looks like.

**Comment by lunaris** [comment]
Pretty sure we have this now (e.g. if I type `y`, I get filtered to `yes`) -- closing as fixed.
